DOCS/xml-fields.md

```diff
@@ -44,17 +44,19 @@ Within the .xml file, the first and last lines of the file should be the followi
 The main tag used to define the license or exception is, unsurprisingly, **`<license>`** or **`<exception>`**. All of the remaining content will be enclosed within a `<license></license>` or `<exception></exception>` pair of tags.
 
 There are two mandatory attributes for every `<license>` and `<exception>` tag:
-* `licenseId`: the unique SPDX Identifier for the license; should be identical to the filename
```

Comment:
Throughout this PR, it looks like in addition to removing the `isDeprecated` documentation, this PR is also changing all of the bullet types from `*` to `-`.

Are these changes needed, since [both bullet types are valid for Markdown lists on GitHub](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists)? 

If not needed, could you please revise the PR so that the only change is removing the `isDeprecated` documentation? Would prefer to keep the edits limited so that it's clear what's being changed here.